pkgs/applications/window-managers/newm/default.nix

```
@@ -0,0 +1,51 @@
+{ lib,
+fetchFromGitHub,
```

Each line should start with comma, not end with it.

```suggestion
{ lib
, fetchFromGitHub
```

And do the same below